README.md

```diff
@@ -16,7 +16,7 @@ subject to change, and some features are not fully implemented.
 ## Certification Workflow Guide
 
 For the complete container and operator bundle certification workflow instructions, please reference our 
-[official certification documentation](https://access.redhat.com/documentation/en-us/red_hat_software_certification/8.56/html/red_hat_software_certification_workflow_guide/index).
+[official certification documentation](https://access.redhat.com/documentation/en-us/red_hat_software_certification/8.69/html/red_hat_software_certification_workflow_guide/index).
```

Comment:
I won't keep this from going in, but is there instead a 'latest' URL that we can use in preflight, so that it does not require a code change and a release in order for these to be updated?

Reply:
Hi @bcrochet there is no 'latest' URL, each URL has got the new version(8.6x). It'd be great to not have the version in the URL then we won't have to update it when there is a new version which is every 3-4 weeks.

Reply:
We might want to figure out a redirector then that is external to preflight.